public/game/views/app-view.coffee

```diff
@@ -8,6 +9,7 @@ AppView = (context) ->
 		nxt.Binding context.view, (phase) ->
 			switch phase
```

Comment:
I did think about it but still have small doubts... Perhaps it's better to add all required views first and refactor